### ISSUE TITLE
DAC 35 - Low Priority WCAG Level AAA - Links (New Window)

### ISSUE THREAD

**Issue by JadeneAd** [ONSvisual/income-scrolly#35]
<img width="499" alt="DAC_Links (New Window)" src="https://user-images.githubusercontent.com/75850106/155573553-c0423f22-842e-464d-afa6-620ce85bc1e6.png">

### Issue
The ‘Open government licence v3.0’ link opens a new tab without warning the user beforehand. Inexperienced screen reader users that click the link will find that the back button of the browser becomes unavailable, and this could lead to disorientation. There are other links on the page that open new tabs. All links that open in new tabs should be checked to ensure that they state this will happen.

<img width="600" alt="DAC 35 Current code ref" src="https://user-images.githubusercontent.com/75850106/155573677-9b15730e-f08f-49e8-86d2-2ad41e5d1d01.png">

**Screen reader comments:**
“I activated the link with text ‘Open Government Licence v3.0’ and was disconcerted to find that the target page had opened in a new tab, even though I received no warning that this would happen. Such behaviour could be problematic for inexperienced assistive technology users who would be unable to return to their previous page via the browser’s ‘Back’ button, and therefore may end up being stuck on an unfamiliar page.”

### Suggested Solution
We advise that links that open a new page alert the user beforehand. This can be achieved with text like (Opens new tab).

<img width="611" alt="DAC 35-2 Current code ref" src="https://user-images.githubusercontent.com/75850106/155573762-7cf06dfe-bd19-4f9d-979f-d9baaf566e44.png">
